dev/src/generate_sql_schema.py

```diff
@@ -43,6 +43,12 @@ class SQL_Delete_Update_Options(str, Enum):
     NO_ACTION = "NO ACTION"
 
 
+class FIELD_SQL_ERROR_ENUM(Enum):
```

Comment:
I assume this differentiates between actual fields and calculated fields? Then I would prefer a more articulate name, e.g. `FieldType`. Listing the properties in the name is not a good naming pattern.

Also, you seem to use different casings for the differrent enums (upper vs. lower snake case etc.), you should use a uniform naming pattern there

Reply:
Used the CapWords or CamelCase style, recommended by PEP8 for type variables and class names.